src/libs/fileDownload/getImageResolution.ts

```diff
@@ -20,14 +21,14 @@ function getImageResolution(file) {
         const objectUrl = URL.createObjectURL(file);
         image.onload = function () {
             resolve({
-                width: this.naturalWidth,
-                height: this.naturalHeight,
+                width: image.naturalWidth,
+                height: image.naturalHeight,
```

Comment:
```suggestion
                width: (this as HTMLImageElement).naturalWidth,
                height: (this as HTMLImageElement).naturalHeight,
```

This will do @fvlvte :)